docs/workspace/advanced_tasks.md

```diff
@@ -61,6 +61,14 @@ build = { cmd = "ninja -C .build", depends-on = ["configure"] }
 start = { cmd = ".build/bin/sdl_example", depends-on = ["build"] }
 ```
 
+You can also specify a specific environment to use for a dependent task:
```

Comment:
Make sure the text makes still sense as a whole. Maybe move it to the end of the section